static/src/js/components/Timeline/Timeline.js

```diff
@@ -266,6 +266,27 @@ export const Timeline = ({
       data.push(dataValue)
     })
 
+    Object.keys(intervals).forEach((conceptId, index) => {
```

Comment:
Double check this I think you need to check which path we are on `pathname: '/search/granules'` is on the focused collection view and pathname: '/project' is on the project view. Right now its going to try to run both workflows. this one and the one above it

Reply:
Yeah that's probably a better solution. Currently it doesn't actually run on both workflows because it's checking to see if the current focused collection has already been added to to `projectCollectionIds` and skipping it if it's already there, but that's probably a more clear workflow.